test/e2e/petset.go

```diff
@@ -408,7 +403,7 @@ var _ = framework.KubeDescribe("Stateful Set recreate", func() {
 	petPodName := "web-0"
 
 	BeforeEach(func() {
-		framework.SkipIfMissingResource(f.ClientPool, StatefulSetGroupVersionResource, f.Namespace.Name)
+		framework.SkipUnlessProviderIs("gce", "vagrant")
```

Comment:
Looks like they're running in GKE, under https://k8s-testgrid.appspot.com/google-gke#gci-gke. So, we should continue to have it enabled.